### DISCUSSION TITLE
Trouble setting up SMTP in docker container

### DISCUSSION THREAD

**Discussion by bigmoof** [requarks/wiki#6166] [Help / Questions] [answered]
OS: Ubuntu 22.04.1 LTS and Docker
Wiki.js 2.5.296
Database engine: PostgreSQL 11

Here is sample docker-compose file :
version: "3"
services:

  postgres-db:
    image: postgres:11-alpine
    environment:
      POSTGRES_DB: wiki
      POSTGRES_PASSWORD: wikijsrocks
      POSTGRES_USER: wikijs
    ports:
      - "5432:5432"
    restart: unless-stopped
    volumes:
      - './postgresql:/var/lib/data'
      - db-data:/var/lib/postgresql/data

  wiki:
    image: ghcr.io/requarks/wiki
    depends_on:
      - postgres-db
    environment:
      DB_TYPE: postgres
      DB_HOST: postgres-db
      DB_PORT: 5432
      DB_USER: wikijs
      DB_PASS: wikijsrocks
      DB_NAME: wiki
    restart: unless-stopped
    ports:
      - "88:3000"
    extra_hosts:
      - "smtp:xx.xx.xx.xx"

volumes:
  db-data:

Using admin page to setup mail, it just keeps giving me "The mail configuration is incomplete or invalid."
I'm just trying to use local docker host or external smtp relay host to test.
After adding extra_hosts, I was able to ping the host from the container.

What am I missing? How can I troubleshoot this? Is there an env that I can use to set this up?

Thanks!

**Comment by florian30491** [comment, marked as answer]
Have you tried to apply your changes before ? 

![image](https://user-images.githubusercontent.com/58624223/225562977-a4fe7426-bcd1-4bd4-90dd-bae343b6eef5.png)

**Reply by bigmoof** [reply]
So silly of me....that works!

**Reply by florian30491** [reply]
I had the same issue 😂
